usermods/PIR_sensor_switch/usermod_PIR_sensor_switch.h

```diff
@@ -299,8 +299,8 @@ void PIRsensorSwitch::publishHomeAssistantAutodiscovery()
     JsonObject device = doc.createNestedObject(F("device")); // attach the sensor to the same device
     device[F("name")] = serverDescription;
     device[F("ids")]  = String(F("wled-sensor-")) + mqttClientID;
-    device[F("mf")]   = "WLED";
-    device[F("mdl")]  = F("FOSS");
+    device[F("mf")]   = WLED_BRAND;

```

Comment:
This may break existing set-ups (and MQTT integrations). It is not needed for the purposes of determining what configuration the binary was built with.

I would avoid changing MQTT configurations if possible.

Reply:
MQTT configuration doesn't change for WLED. It stays mf = WLED and mdl = FOSS. 

This allows people who **do not want** mqtt to return WLED and FOSS. 
Also, this is in a usermod. 

You don't need to worry about this breaking anything, because nothing WLED will ever change this value. 
People who make custom build of WLED were already able to modify this value if they wished. I am just making it easier to do so. 

If someone had a custom build that already had a custom brand set, this will not break their integration. 
If someone had a normal build, this will not break any integration because it is the same value. Nothing changed for them.